lms/models/application_instance.py

```diff
@@ -202,7 +202,7 @@ def lti_version(self) -> str:
         The return values (LTI-1p0, "1.3.0) are the same the spec defines
         and will match the version parameter on lti launches.
         """
-        if self.lti_registration_id:
+        if self.lti_registration_id and self.deployment_id:
```

Comment:
We should never find this one without the other in our DB but this is more accurate.